docs/reference/content/guides/annotations.md

```diff
@@ -0,0 +1,292 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Annotations"
+[menu.main]
+  parent = "Reference Guides"
+  weight = 85
+  pre = "<i class='fa fa-file-text-o'></i>"
++++
+
+# Annotations
+
+Below is a list of all the annotations and a brief descriptions of how to use them.
+
+## Entity
+Marks entities to be stored directly in a collection. This annotations is optional in most cases (though this is likely to change in 
+future versions). There is no harm in including it to be more verbose, and make clear the intention for the class.  The definition of 
+this annotation looks like this:
+
+```java
+public @interface Entity {
+  String value() default Mapper.IGNORED_FIELDNAME;
+  CappedAt cap() default @CappedAt(0);
+  boolean noClassnameStored() default false;
+  boolean queryNonPrimary() default false;
+  String concern() default "";
+}
+```
+| Parameter           | Usage           |
+| ------------------- | --------------  |
+| value()             | Defines the collection to use.  Defaults to using the classname |
+| cap()               | Marks this collection as capped and sets the size to use.  See the [`@Capped`]({{< ref "#capped" >}}) below|
+| noClassnameStored() | Tells Morphia to not store the classname in the document.  The default is to store the classname. |
+| queryNonPrimary()   | Indicates that queries against this collection can use secondaries.  The default is primary only reads. |
+| concern()           | The WriteConcern to use when writing to this collection. |
+
+
+## Indexes
+
+In addition to being able to declare an index on a single field you can also declare the indexes at the class level. This allows you to 
+ create more than just a single field index; it allows you to create compound indexes with multiple fields.
+
+```java
+public @interface Indexes {
+    Index[] value();
+}
+```
+
+### Index
+ 
+```java
+public @interface Index {
+    String value() default "";
+    String name() default "";
+    boolean unique() default false;
+    boolean dropDups() default false;
+    boolean background() default false;
+    boolean sparse() default false;
+    boolean disableValidation() default false;
+    int expireAfterSeconds() default -1;
+
+    Field[] fields() default {};
+    IndexOptions options() default @IndexOptions();
+}
+```
+
+There are two pieces to this annotation that are mutually exclusive.  The first group of parameters are considered legacy.  They are safe
```

Comment:
This seems wrong.  Most of the first group (unique, dropDups, etc) are not part of Field nor would it make sense for them to be.